composer.json

```diff
@@ -35,7 +35,7 @@
         "laminas/laminas-mvc-plugin-identity": "^1.2",
         "laminas/laminas-mvc-plugin-prg": "^1.3"
     },
-    "replace": {
-        "zendframework/zend-mvc-plugins": "^1.0.1"
+    "conflict": {
+        "zendframework/zend-mvc-plugins": "*"
```

Comment:
Also here: weird that this replaces the `zendframework/` version without providing the bridge.

The diff is correct though 👍